### PR TITLE
TMA-1205: Fix vcr configurer setup

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: spec/vcr_configurer.rb [hunk 1]
```diff
@@ -10,6 +10,56 @@ class VcrConfigurer
...
+          end
+
+          vcr_config.default_cassette_options = {
+            :decode_compressed_response => true,
+            :match_requests_on => [gdc_path_matcher, :method, :query],
+            # allow to set record mode from environment, see https://relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes
+            :record => vcr_record_mode
+          }
+
+          %w(request response).each do |part|
```
Comment (yenkins) [inline spec/vcr_configurer.rb]: info: Contains iterators nested 2 deep (NestedIterators) (reek)

Comment (yenkins) [inline spec/vcr_configurer.rb]: warning: Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`. (rubocop)

File: spec/vcr_configurer.rb [hunk 2]
```diff
@@ -10,6 +10,56 @@ class VcrConfigurer
       VCR_PROCESS_ID = 'VCRFakeProcessId'
       @ignore_vcr_requests = false
 
+      def self.setup
```
Comment (yenkins) [inline spec/vcr_configurer.rb]: info: Has approx 24 statements (TooManyStatements) (reek)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/142/ : SUCCESS in 12s

**Pull request merged**